### DISCUSSION TITLE
Procurement support?

### DISCUSSION THREAD

**Discussion by poVoq** [baimard/gestion#259] [Ideas]
A lot of small orgs (espcially NGOs) have to follow public procurement procedures, such as requesting multiple quotes (RFQ/RFP) or publicize a tender etc.

Would be cool if there was a simple way to track purchase requests and allow bidders to upload documents on a unique link etc.

**Comment by baimard** [comment]
I'm sorry I don't understand, could you explain without acronym ?

And you're publicize tender or answering a tender ?

**Comment by poVoq** [comment]
Publicizing. Although tender is probably too big. Rather requesting companies to submit their offers in a standardized and easy to manage way.

**Reply by baimard** [reply]
I'm not sur if Gestion is the good application for your use case.

**Comment by les-les** [comment]
sorry but to be honest no it is not. 

gestion is on its way (if the development team follows the lead ) to be a great app for 

bid / offert / device 
and
invoice 

maybe with an build in tracking (especially for bids / devis ) 
and a tracking for invoices, both easy to handle and no rocket science 

and that's the scope 
get this runnin perfect and the rest will follow. 

procurement is a total different ballpark !

br LES
